staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -188,10 +189,28 @@ func conditionFuncFor(condition string, errOut io.Writer) (ConditionFunc, error)
 			conditionName = conditionName[0:equalsIndex]
 		}
 
+		conditions[conditionName] = conditionValue
 		return ConditionalWait{
-			conditionName:   conditionName,
-			conditionStatus: conditionValue,
-			errOut:          errOut,
+			conditions: conditions,
+			errOut:     errOut,
+		}.IsConditionMet, nil
+	}
+
+	if strings.HasPrefix(condition, "conditions=") {
```

Comment:
I'm assuming that's a newly introduced format, is this really necessary? Can you post an example somewhere?

Reply:
Also the code ends-up being repeated here

Reply:
My bad. Here's an example of how this would look like: 
`kubectl wait--for=conditions=Available=False||Progressing=False --timeout=20s deployments/nginx-app`
So the reason for having a new format while maintaining the previous one is to maintain backwards compatibility and avoid creating a new flag.

Reply:
I'm sorry if I wasn't clear. I was suggesting we do what you suggest like this:
```
# Waits for one of the condition to be true:
kubectl wait --for=condition=Available=False --for=condition=Progressing=False
# Run multiple kubectl wait command in a row if you want to do a AND
kubectl wait --for=...
```